pyeudiw/openid4vp/direct_post_response.py

```diff
@@ -146,7 +147,7 @@ def get_presentation_vps(self) -> list[Vp]:
 
         for vp in vps:
             # TODO - add an exception handling here
-            _vp = Vp(vp)
+            _vp = handle_VP(vp)
```

Comment:
non è un nome di compliant alla PEP, rinominalo in vp_verifier, vp_parser o simile